### PR TITLE
Add/small cleanup

### PR DESCRIPTION
**Pull request by millerf**
#### Changes proposed in this Pull Request

* Added a README for the Sift webhooks
* Fix name of a key for the webhook endpoint
* Remove unecessary file (`abuse-decisions-action.php`)
* Change file hierarchy and harmonize file/class names

#### Testing instructions

* Make sure everything works.

### CODE REVIEW

File: tests/PaymentGatewayTest.php [hunk 1]
```diff
@@ -4,7 +4,7 @@
  */
 declare( strict_types=1 );
 
-use Sift_For_WooCommerce\Payment_Gateway;
+use Sift_For_WooCommerce\Sift_Payment_Gateway;
 
 require_once __DIR__ . '/../src/inc/payment-gateway.php';
```
Comment (chrisfromthelc) [inline tests/PaymentGatewayTest.php]: I think this line needs to be removed/changed.

**Review by chrisfromthelc — CHANGES_REQUESTED** [review]
Looks like some linter issues and a bad file requirement in the payment gateway test.

**Review by chrisfromthelc — DISMISSED** [review]

**Review by millerf — COMMENTED** [review]
LGTM. But I cannot approve.

**Review by chrisfromthelc — APPROVED** [review]
Approving for @millerf

**Pull request merged**